### ISSUE TITLE
Hammerting Demo (1316980)

### ISSUE THREAD

**Issue by brandonegbert** [ValveSoftware/Proton#4002]
# Compatibility Report
- Name of the game with compatibility issues: Hammerting Demo
- Steam AppID of the game: 1316980

## System Information
- GPU: Radeon 5700 XT
- Driver/LLVM version: Mesa 20.1.1 / LLVM 10.0.0
- Kernel version: 5.7.2
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/brandonegbert/bec844b6d8cd3c632422ff5c67cf5ea3
- Proton version: 5.0-9

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1316980.log](https://github.com/ValveSoftware/Proton/files/4802001/steam-1316980.log)

## Symptoms 
* About half the time the splash screen and main menu never appear though the mouse shows up and audio works.
* Sometimes the splash screen and UI elements appear but flicker at a high rate, and sometimes they work fine.
* Extreme artifacting once entering a game and mousing over certain gameplay elements.  Looks fine when not doing this.  I suspect this is an outline that is supposed to appear around the objects.
![Screenshot_20200618_184001](https://user-images.githubusercontent.com/11236365/85087831-6a4d9680-b193-11ea-820a-800364ca2ecd.png)
* Entire graphics driver seems to crash after around 40 seconds.  Game sound still runs and switching to another TTY works but the screen no longer updates and alt tabbing no longer works.  Mouse still moves though.

## Reproduction
Launch the game and try to start a new game.

**Comment by kisak-valve** [comment]
Hello @brandonegbert, can you check if setting the game's launch options to `RADV_PERFTEST=aco %command%` has an effect on the rendering and stability issues?

**Comment by brandonegbert** [comment]
No, it's not set.  Just retested with RADV_PERFTEST=llvm to be sure and still had it behave the same, though the artifact looked different (could be coincidence).
![Screenshot_20200618_212547](https://user-images.githubusercontent.com/11236365/85097070-2feff380-b1ab-11ea-8715-1091f340047a.png)

**Comment by kisak-valve** [comment]
With mesa 20.1.1, LLVM is RADV's default shader compiler and `RADV_PERFTEST=llvm` doesn't do anything (the same as having nothing set). I was asking if the ACO shader compiler has different behavior from the LLVM default, not if you were using it in your initial test.

**Comment by brandonegbert** [comment]
Ah, sorry about that.  Same result with ACO.

**Comment by brandonegbert** [comment]
Seems to work great now with 5.13-1.
